lib/lmi_for_all.rb

```diff
@@ -0,0 +1,61 @@
+class LmiForAll
+  def initialize(soc_code)
+    @soc_code = soc_code
```

Comment:
Why not just pass in the `soc_code` on the lookup method? (Then we can avoid giving this class any state)